df_py/volume/test/test_calc_rewards.py

```diff
@@ -752,3 +756,90 @@ def mock_multipliers(DF_week, is_predictoor):  # pylint: disable=unused-argument
         assert rewards_info[C1][NA][LP1] == approx(60.3, abs=1e-5)
         assert rewards_info[C2][NB][LP2] == 300
         assert rewards_info[C2][NB][LP3] == 300
+
+
+@enforce_types
+def test_volume_reward_calculator_pdr_boost(tmp_path):
```

Comment:
Change this test such that it tests both DF_week=81 and DF_week=82. See comment above.

How I'd do it: 
```python
@enforce_types
def test_volume_reward_calculator_pdr_boost_week81(tmp_path):
    _test_volume_reward_calculator_pdr_boost(tmp_path, DF_week=81):

@enforce_types
def test_volume_reward_calculator_pdr_boost_week82(tmp_path):
    _test_volume_reward_calculator_pdr_boost(tmp_path, DF_week=82):

@enforce_types
def _test_volume_reward_calculator_pdr_boost(tmp_path):
    ...
    # (like your implementation, with diff't expected numbers for week 81 vs 82)
```

(I prefer two test functions instead of parameterizing a single test function, when there's only a couple cases. Easier to work with.)